benchmark/benchmarks.dart

```diff
@@ -4,46 +4,117 @@ import 'package:benchmark_harness/benchmark_harness.dart';
 
 import 'package:r_tree/r_tree.dart';
 
-final int BRANCH_FACTOR = 16;
-
+final int branchFactor = 8;
+final int randomSeed = 3;
```

Comment:
Used a common seed here so the tests all run against the same data sets and makes re-running more predictable.